app/views/commodityCodeBestMatch.scala.html

```diff
@@ -35,10 +35,12 @@
 
         @components.input_yes_no(
             field = form("value"),
-            labelHtml = Some(components.heading(messages("commodityCodeBestMatch.heading"))),
+            labelHtml = Some(components.heading(
+                messages("commodityCodeBestMatch.heading", goodsName),
+                captionMsg = Some(messages("commodityCodeBestMatch.caption")))),
             hint = Some(messages("commodityCodeBestMatch.bodyText"))
         )
 
-        @components.submit_button()
+        @components.submit_button(Some(messages("commodityCodeBestMatch.submitLabel")))
```

Comment:
Just noticed that we should probably use "site.save_and_continue"

Reply:
Yeah we were talking about this a bit yesterday - we thought we could go through the site when we did the navigation part and change all of the pages to use it